internalshared/configutil/config.go

```diff
@@ -167,16 +167,27 @@ func (c *SharedConfig) Sanitized() map[string]interface{} {
 	}
 
 	result := map[string]interface{}{
-		"disable_mlock": c.DisableMlock,
-
+		"cluster_name":                 c.ClusterName,
 		"default_max_request_duration": c.DefaultMaxRequestDuration,
+		"disable_mlock":                c.DisableMlock,
+		"log_format":                   c.LogFormat,
+		"log_level":                    c.LogLevel,
+		"pid_file":                     c.PidFile,
+	}
 
-		"log_level":  c.LogLevel,
-		"log_format": c.LogFormat,
-
-		"pid_file": c.PidFile,
-
-		"cluster_name": c.ClusterName,
+	// Optional log related settings
+	switch {
```

Comment:
❓ why use a switch statement with fallthrough here, rather than multiple if statements?

Reply:
I think I'm just having a renaissance with `switch` at the moment, and I wanted a way to make them feel grouped together vs. separate `if {}` ... `if {}` ...  blocks.

Reply:
I should add I'm happy to change it over if you think it might read easier :)

Reply:
I do think it's a bit easier to read, and it decreases the chances that we'll add a case in the future and forgot to put `fallthrough`

Reply:
That's what the tests are there for 😄 (RE: `fallthrough` worry).

I'll push up a change now 👍🏼